buildpack/core/java.py

```diff
@@ -305,6 +305,13 @@ def _set_jvm_memory(m2ee, vcap):
     heap_size = str(heap_size) + "M"
 
     env_heap_size = os.environ.get("HEAP_SIZE")
+    max_metaspace_size = os.environ.get("MAX_METASPACE_SIZE")
+    
+    if max_metaspace_size:
+        util.upsert_javaopts(m2ee, "-XX:MaxMetaspaceSize=%s" % max_metaspace_size)
+    else:
+        util.upsert_javaopts(m2ee, "-XX:MaxMetaspaceSize=256M")
+        
```

Comment:
This code fragment can be changed into a single line by using the `os.getenv` function: https://docs.python.org/3/library/os.html#os.getenv. The second parameter can be used to assign a default value if the environment variable is unspecified